.travis-build.sh

```diff
@@ -7,6 +7,15 @@ bootstrap_header() {
   echo -e "${ANSI_YELLOW}$1${ANSI_RESET}"
 }
 
+travis_fold() {
```

Comment:
> Going back to a "travis.yml only" setup would be my preference...

i tried it, but i didn’t find a way to get folding then. separate shell lines in YAML are treated as separate commands, which messes with the folding, …

maybe we can put it all in the YAML like this?

```yaml
script:
  - |
    travis_fold start Building
    bootstrap_header 'Building package'
    R CMD build .
    travis_fold end Building
  - ...
```